edgedb/protocol/codecs/edb_types.pxi

```diff
@@ -27,4 +27,5 @@ TYPE_IDS = {
     'std::duration': uuid.UUID('00000000-0000-0000-0000-00000000010e'),
     'std::json': uuid.UUID('00000000-0000-0000-0000-00000000010f'),
     'std::bigint': uuid.UUID('00000000-0000-0000-0000-000000000110'),
+    'std::relativedelta': uuid.UUID('00000000-0000-0000-0000-000000000111'),
```

Comment:
```suggestion
    'cal::relativedelta': uuid.UUID('00000000-0000-0000-0000-000000000111'),
```

Reply:
Should I also change `std::local_datetime` etc. to `cal`?

Reply:
Yes, that was an oversight.

Reply:
changed